src/backend/users/management/commands/migrate_housing_seeker_data.py

```diff
@@ -36,8 +36,9 @@ class LocationKeys:
 
 
 class PositionKeys:
-    LON = "lon"
-    LAT = "lat"
+    # swap lon & lat fields to account for mislabled profile data
```

Comment:
This does fix the problem but I don't think that this is an accurate characterization of the issue here -- if you look at the latitude and longitude values in one of the profiles, they do match the real-world latitudes and longitudes of the Boston area. However, on [line 394](https://github.com/azavea/echo-locator/blob/develop/src/backend/users/management/commands/migrate_housing_seeker_data.py#L394) when we're creating the geometry, we're doing it with `Point(x=lat, y=lon)`, which isn't correct (latitude is North/South, or the Y direction, while longitude is East-West, or X). I think I probably missed this in PR review, so apologies to @aaronxsu ! But in any case, I think changing to `Point(x=lon, y=lat)` is the right way to go here.

Reply:
@ddohler I think I see where the confusion may come from. So actually on [these lines](https://github.com/azavea/echo-locator/blob/develop/src/backend/users/management/commands/migrate_housing_seeker_data.py#L384-L385), `lat` and `lon` pull from the corrected property keys from the user profile json object, since they were flipped in this `PositionKeys`. 

An alternative fix would be to not flip the `PositionKeys`, but rather flipping how we pass arguments when creating `Point`.

Reply:
I do think for better readability and for future us to reason the semantic of these variable names, maybe flipping how we create `Point` is better than flipping how we handle original keys in the object.

Reply:
> An alternative fix would be to not flip the PositionKeys, but rather flipping how we pass arguments when creating Point.

Yep, I think that's the right way to solve this! :+1:

Reply:
The latest commit I forced push to this PR implements the change as discussed by reversing the inputs when the `Point` in created. Also I rebased this branch on top of the latest version of `develop`.  We successfully performed the migration on staging again using this new commit.

Reply:
Good job. I believe this should be good to go!